_maps/icebox.json

```diff
@@ -3,7 +3,7 @@
 	"map_name": "Ice Box Station",
 	"map_path": "map_files/IceBoxStation",
 	"map_file": [
-		"IcemoonUnderground_Below_skyrat.dmm",
+		"IcemoonUnderground_Below.dmm",
```

Comment:
Is this intentional?

Reply:
Yeah it is, that map wasn't touched, if we wanna make it modular, we can.